docs/api/ethereum.md

```diff
@@ -1,5 +1,645 @@
+| handler | [required] A callback `function` which is triggered on each `event`. When the `event` equals `ACCOUNT_CHANGE`, the first argument is a new account ID and the second is the old one, when the `event` equals `NETWORK_CHANGE`, the first argument is a new network version and the second is the old one.
+
+**Result:**
+
+An instance of the same provider class.
+
+**Example:**
+
+```ts
+import { ProviderEvent } from '@0xcert/ethereum-bitski-backend-provider'; // or another provider
+
+provider.on(ProviderEvent.NETWORK_CHANGE, (networkVersion) => {
+  console.log('Network has changed', networkVersion);
+});
+```
+
+**See also:**
+
+[once (event, handler)](#once-event-handler), [off (event, handler)](#off-event-handler)
+
+### once(event, handler);
+
+A `synchronous` class instance `function` which attaches a new event handler. The event is automatically removed once the first `event` is emitted.
+
+**Arguments:**
+
+| Argument | Description
+|-|-
+| event | [required] A `string` representing a [provider event](./ethereum.md#provider-events) name.
+| handler | [required] A callback `function` which is triggered on each `event`. When the `event` equals `ACCOUNT_CHANGE`, the first argument is a new account ID and the second is the old one, when the `event` equals `NETWORK_CHANGE`, the first argument is a new network version and the second is the old one.
+
+**Result:**
+
+An instance of the same provider class.
+
+**Example:**
+
+```ts
+import { ProviderEvent } from '@0xcert/ethereum-bitski-backend-provider';
+
+provider.on(ProviderEvent.NETWORK_CHANGE, (networkVersion) => {
+  console.log('Network has changed', networkVersion);
+});
+```
+
+**See also:**
+
+[on (event, handler)](#on-event-handler), [off (event, handler)](#off-event-handler)
+
+### off(event, handler)
+
+A `synchronous` class instance `function` which removes an existing event handler.
+
+**Arguments:**
+
+| Argument | Description
+|-|-
+| event | [required] A `string` representing a [provider event](./ethereum.md#provider-events) name.
+| handler | A specific callback `function` of an event. If not provided, all handlers of the `event` are removed.
+
+**Result:**
+
+An instance of the same provider  class.
+
+**Example:**
+
+```ts
+import { ProviderEvent } from '@0xcert/ethereum-bitski-backend-provider';
+
+provider.off(ProviderEvent.NETWORK_CHANGE);
+```
+
+**See also:**
+
+[on (event, handler)](#on-event-handler), [once (event, handler)](#once-event-handler)
+
+### orderGatewayId
+
+A class instance `variable` holding a `string` which represents an Ethereum address of the [order gateway](/#public-addresses).
+
+### requiredConfirmations
+
+A class instance `variable` holding a `string` which represents the number of confirmations needed for mutations to be considered confirmed. It defaults to `1`.
+
+### signMethod
+
+A class instance `variable` holding a `string` which holds a type of signature that will be used (e.g. when creating claims).
```

Comment:
`A class instance `variable` holding a `string` which holds a type of signature that will be used (e.g., when creating claims).`